### PR TITLE
sort tags in filter

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>

* Resolves: #1700
* Target version: master

### CODE REVIEW

**Comment by juliushaertl** [comment]
Mind to tackle the other ones from https://github.com/nextcloud/deck/issues/1700#issuecomment-688202834 as well?

- [ ] label listing on cards
- [ ] label list in the sidebar multiselect
- [ ] label list in the sidebar multiselect autocompletion

**Comment by stefan-niedermann** [comment]
> label list in the sidebar multiselect autocompletion

The Android apps sorts in the autocompletion by "most often used in this project" - what do you think about this?

**Comment by juliushaertl** [comment]
> The Android apps sorts in the autocompletion by "most often used in this board" - what do you think about this?

Would be a good next step I'd say.

File: src/components/card/CardSidebar.vue [hunk 1]
```diff
@@ -389,7 +392,8 @@ export default {
 			}
 
 			this.copiedCard = JSON.parse(JSON.stringify(this.currentCard))
-			this.allLabels = this.currentCard.labels
+			// this.allLabels = this.currentCard.labels
```
Comment (juliushaertl) [inline src/components/card/CardSidebar.vue:395]: :broom:

File: src/components/card/CardSidebar.vue [hunk 2]
```diff
@@ -389,7 +392,8 @@ export default {
...
+			this.allLabels = [...this.currentCard.labels].sort((a, b) => (a.title < b.title) ? -1 : 1)
```
Comment (juliushaertl) [inline src/components/card/CardSidebar.vue:396]: I think the naming is now a bit confusing, allLabels should rather be assignedLabels.

File: src/components/cards/CardItem.vue [hunk 3]
```diff
@@ -122,6 +123,10 @@ export default {
 		currentCard() {
 			return this.card && this.$route && this.$route.params.cardId === this.card.id
 		},
+		
+    () {
```
Comment (juliushaertl) [inline src/components/cards/CardItem.vue:127]:
```suggestion
		labelsSorted() {
``` ?

Comment (jakobroehrl) [reply src/components/cards/CardItem.vue]: I don't know what the problem here is. In my IDE it's correct:
grafik

**Comment by juliushaertl** [comment]
Please rebase :)

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**